src/C1T2X_OBU.py

```diff
@@ -137,7 +141,20 @@ def VANET_listening_thread():
 			c1t2x_logger.debug("Received %s from VANET", pkt)
 			if pkt:
 				if not parseVANETPacket:
-					sendLAN(pkt[0])
+					# Check if ack or payload
+					data = pkt[0].decode('utf-8')
+					if data == "1":  # Ack
+						with mutex:
+							wait_for_ack = False
+						c1t2x_logger.info("Received ack")
+					elif data == previous_packet_received:  # Duplicate message received, so just resend ack
```

Comment:
More of a general comment. Do we want to add ACKs to LAN as well? If it's being reliable and it's just wifi that's the issue then maybe we're fine without.

Reply:
Acks to the LAN would required modifying V2X Hub and CARMA Messenger, so I think sticking with wifi should be fine here.